src/main/play-doc/operation/Licensing.md

```diff
@@ -0,0 +1,107 @@
+- Components of Enterprise Suite that your license is authorized for.
+
+### Offline Mode
+
+In privileged environments, loading the license is straightforward.  In environments that cannot access the internet you 
+must install the license in offline mode (i.e. from your CLI without having the HTTP proxy setup, or if you’re 
+installing the CLI on a server with no internet access).
+
+Steps to install the license in offline mode.
+
+1. Login to your lightbend.com account with your licensed email and setup your `~/.lightbend/commercial.credentials` 
+as described on [lightbend.com](https://www.lightbend.com/product/conductr/developer).  While setting up your 
+`commercial.credentials` it’s important to use the email associated with your organization's license, and not a personal
+account.  If enterprise environments this is usually your corporate email.
+2. Download your `license` file manually.  You can find your license by visiting 
+https://www.lightbend.com/product/conductr/license
+3. Copy & paste the contents of this page into your `~/.lightbend/license` file.
+4. Execute `conduct load-license --offline` at the command line.
+
+```
+$ conduct load-license --offline
+Skipping downloading license from Lightbend.com
+Loading license into ConductR at 192.168.10.1
+
+Licensed To: aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa
+Expires In: 365 days (Fri 27 Jul 2018 16:09PM)
+Grants: akka-sbr, cinnamon, conductr, fortify
+
+License successfully loaded
+```
+
+### Viewing the current license ConductR cluster
+
+When you issue `conduct info` you will see the same license details that were returned when the license was first
+loaded.
+
+### Updating your License
+
+If you need to update the license on an already licensed cluster you can do so using the `--force` command switch. This
+is useful when you want to change the license to a different user or when your license is close to expiring.  Enter
+`conduct load-license --offline` and the command line and then follow the prompts to load a new license.
+
+## Automate Licensing at Startup
+
+When a license is loaded it is submitted to the ConductR cluster and distributed to all the ConductR core nodes.  The
+license resides in memory, so in the event of a complete cluster shutdown the data is lost.  In production this would
+be an unlikely scenario as your cluster should be able to tolerate the failure of some nodes.
+
+In non-production environments it may be commonplace to shutdown whole clusters for a number of reasons: cluster is no
+longer required, to save on public cloud instance charges, etc.
+
+In any environment, whenever a cluster starts up it is important to license before bundles are loaded and run again.
+By retrieving your license manually as described in "Offline Mode", you can automate the licensing of a cluster whenever
+it's first started up.
+
+Remember that licenses expire, so set a reminder to occassionally retrieve an updated `./.conductr/license` file for 
+your deployment script.
```

Comment:
@seglo I don't see/know of anything that clearly would prevent the new scale request from being fulfilled, provided said agents have sufficient resources in their offers. We want you to be successful and we have no motivation to stop existing bundles in the future, for example. Beyond that it is difficult to provide warranty for future behavior.